My recent work section

lily-coder opened the pull request:
This is the middle section of the portfolio webpage. I have worked of the following sections:

- The multi-post stories section
- The ellipses
- The social icons section

Shaher-11 commented on index.html:
```diff
@@ -53,5 +53,81 @@ <h1 class="jong">Lilian Moraa</h1>
+                <p class="white">bootstrap</p> <hr>
+                <p class="white">ruby</p>
+            </div>
+            <div class="story-btn1"><button type="button" class="btn1 bgb white">See Project</button></div>
```
Style the btn as shown in the original design
![right1](https://user-images.githubusercontent.com/58376020/124969897-8be02100-e027-11eb-9e56-448a4701146d.png)

Shaher-11 reviewed (changes requested):
#### Hello @lily-coder,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Update the readme file with project info, project screenshot, live demo.
- [x] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 remaining reviews. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

lily-coder commented:
- I have updated the README file with project info, project screenshot and live demo.

- I have styled the button as the original

- And I have also fixed the styling for the ellipses section and the text is now visible

Georjane reviewed (approved):
## Status: PR Approved 💯 🥇 
#### Hello @lily-coder , 

Very nice portfolio you got there! Great pick from amongst the Figma template!
Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations! 🎉 👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- following the button styling: Active, Hover, and Pressed for all buttons on your page ([see Figma design](![image](https://user-images.githubusercontent.com/60772732/124997208-20577d00-e042-11eb-8210-1451d8daa1f0.png)))

_Good luck with your next project and Happy coding!_ 💻
